docker.entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Run some command to make cli parse and setup config passed by --pass-config
```

Comment:
I think to properly resolve this hack we have to move `--pass-config` from cli to api of neuromation-client. Should I create an issue?

Reply:
Not yet. I dislike the `--pass-config` implementation actually